### DISCUSSION TITLE
How can I list all tables in an SQLite database using Prisma?

### DISCUSSION THREAD

**Discussion by tanem** [prisma/prisma#3213] [Q&A] [answered]
Hi there,

I've been playing around with the [Simple TypeScript Script Example](https://github.com/prisma/prisma-examples/tree/master/typescript/script), but get no results when I try to list all tables in the DB using Prisma.

I can query the DB directly to get the results I want:

```
$ sqlite3 prisma/dev.db \
'SELECT name FROM sqlite_master WHERE type = "table";'
sqlite_sequence
User
Post
```

But if I use `queryRaw` in `script.ts`, I get an empty result:

```ts
const result = await prisma.queryRaw`SELECT name FROM sqlite_master WHERE type = "table";`
console.log(result);
// Output: []
```

Any ideas?

Thanks 🙏

**Comment by ryands17** [comment]
Hey @tanem 👋 
I can reproduce this, will create an issue for this as well :)

**Reply by ryands17** [reply]
https://github.com/prisma/prisma/issues/3218

**Reply by tanem** [reply]
Thanks @ryands17, I'll follow that issue too 👍

**Comment by tanem** [comment, marked as answer]
Resolved thanks to [this issue comment](https://github.com/prisma/prisma/issues/3218#issuecomment-670473376).

```ts
const result = await prisma.queryRaw`SELECT name FROM dev.sqlite_master WHERE type = "table";`
console.log(result);
// Output: [ { name: 'sqlite_sequence' }, { name: 'User' }, { name: 'Post' } ]
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
